### ISSUE TITLE
md-dialog hidden in 0.11.2

### ISSUE THREAD

**Issue by tonyawad88** [angular/material#5004]
Hello,

I upgraded my material libraries from 0.10.1 to 0.11.2 and the basic mdDialog seems to be hidden. Check screenshot below. I tried to modify the z-index of its container and itself, it didn't work. 

I included below the js and html of the view. 
***HTML Snippet:***
```HTML
<md-content flex style="background-color: transparent;">
...

    <div class="row row-items paperbg">

        <form role="form" name="advwzus" class="col-md-12 aditemgrp">

            <md-input-container flex md-no-float>
                <input ng-model="user.phone" md-maxlength="20" type="text" placeholder="{{lblPhoneNumber}}">
            </md-input-container>

            <md-input-container flex md-no-float>
                <input ng-model="user.email" md-maxlength="100" placeholder="{{lblEmail}}">
            </md-input-container>

            <md-input-container flex md-no-float>
                <textarea ng-model="user.comments" columns="1" md-maxlength="300"
                          placeholder="{{lblComments}}"></textarea>
            </md-input-container>

            <md-button class="md-raised m-0 p-0 m-t-20 m-b-20" ng-disabled="!advwzus.$valid"
                       style="width: 100%;background-color: #d47051;color:#ffffff;"
                       ng-click="showAlert($event)">{{lblSubmit}}
            </md-button>
        </form>


    </div>
    ...
</md-content>
```
***JS Snippet:***
```JS
$scope.showAlert = function (ev) {
        // Appending dialog to document.body to cover sidenav in docs app
        // Modal dialogs should fully cover application
        // to prevent interaction outside of dialog
        $mdDialog.show(
            $mdDialog.alert()
                .parent(angular.element(document.body))
                .title('Thank you !')
                .clickOutsideToClose(true)
                .content('We will be in touch within 24 hours.')
                .ariaLabel('Alert Dialog Demo')
                .ok('Sounds Good !')
                .targetEvent(ev)
        );
        $scope.user = {};

    };
```

Thank you,
<img width="323" alt="screen shot 2015-10-04 at 1 26 50 pm" src="https://cloud.githubusercontent.com/assets/3638322/10269142/a9ae084e-6a9b-11e5-9f70-55be74b9ba1d.png">

**Comment by EladBezalel** [comment]
Please provide a CodePen or Plunkr that demonstrates this issue. Here are some starter demo templates that you can use/fork:

*  [CodePen template](http://codepen.io/team/AngularMaterial/pen/BNYppz)
*  [Plunkr template](http://plnkr.co/edit/yP0gB9iTacIQb48thud8?p=preview)

**Comment by tonyawad88** [comment]
@EladBezalel I wasn't able to replicate using CodePen. However I noticed the opacity of the dialog was set to 0. Apparently we animate the dialog opacity from 0 to 1 when we show it. I have animation disabled. Enabling the animation made it work again. Is it possible to disable animation and still have the dialog show up ?

Hard code css opacity to 1 ?

**Comment by EladBezalel** [comment]
dup #4862,
closed by #5006
